riscos_toolbox/mixins/printing.py

```diff
@@ -51,23 +51,23 @@ def print_event(self, code, id_block, event):
         sideways = event.sideways
 
         try:
-            features = swi.swi("PDriver_Info",";...I")
+            features = swi.swi("PDriver_Info", ";...I")
         except:
             raise RuntimeError("Printer driver not loaded.")
 
-        job = swi.swi("OS_Find","Is;I",0x83,"printer:")
+        job = swi.swi("OS_Find", "Is;I", 0x83, "printer:")
         if job == 0:
             raise RuntimeError("Failed to open printer:")
```

Comment:
These `RuntimeError`s should probably be errors dedicated to the object, so that they can be trapped more usefully.